src/index.html

```diff
@@ -7,7 +7,7 @@
   <title>DSpace</title>
   <meta name="viewport" content="width=device-width,minimum-scale=1">
   <link rel="icon" type="image/x-icon" href="assets/images/favicon.ico" />
-  <link class="theme-css" rel="stylesheet" href="/base-theme.css">
+  <link class="theme-css" rel="stylesheet" href="/dspace-theme.css">
```

Comment:
why this has been replaced with the themed css? Should the theme applied override the base-theme.css automatically?

Reply:
It should, feel free to put it back and verify that it will update automatically. But it's an efficiency thing. 

If you put the css needed for the initial page load there in the first place, it prevents people from having to download the wrong css first, before app.component can replace it.

It is something I would recommend everybody does after they've decided on the theme they want.

Reply:
it makes sense, maybe for the 7.1 we can think to improve it by selecting the css according to the chosen theme. It could be made with webpack, here an article that could help https://medium.com/dailyjs/inserting-variables-into-html-and-javascript-with-webpack-80f33625edc6